buildpacks/_shared/tests/setup.go

```diff
@@ -15,14 +14,13 @@ import (
 )
 
 func NewLogger() bard.Logger {
-	buf := bytes.NewBuffer(nil)
-	return bard.NewLogger(buf)
+	return bard.NewLogger(io.Discard)
```

Comment:
@andrew-su I don't remember what we were using `buf` with, but `io.Discard` does have different behavior.

Reply:
So far, the tests haven't needed the actual log output so it doesn't make a difference for our current use. 

That said, if a test ever wanted to assert on the log contents, they would need to call `bard.NewLogger(buf)` themselves, then make assertions on `buf.String()`. So I gather that this is just a convenience function to create a new logger that satisfies constructors.